perllib/FixMyStreet/Cobrand/IsleOfWight.pm

```diff
@@ -70,6 +70,92 @@ sub open311_munge_update_params {
     $params->{description} = "FMS-Update: " . $params->{description};
 }
 
+sub munge_category_list {
+    my ($self, $options, $contacts, $extras) = @_;
+
+    my $user = $self->{c}->user;
+    my %bodies = map { $_->body->name => $_->body } @$contacts;
+    my $b = $bodies{'Isle of Wight Council'};
+
+    if ( $user && ( $user->is_superuser || $user->belongs_to_body( $b->id ) ) ) {
+        return;
+    }
+
+    @$contacts = grep { $_->send_method && $_->send_method eq 'Triage' } @$contacts;
+    my $seen = { map { $_->category => 1 } @$contacts };
+    @$options = grep { my $c = ($_->{category} || $_->category); $c =~ 'Pick a category' || $seen->{ $c } } @$options;
+}
+
+sub munge_category_where {
+    my ($self, $where) = @_;
+
+    my $user = $self->{c}->user;
+    my $b = $self->{c}->model('DB::Body')->for_areas( $self->council_area_id )->first;
+    if ( $user && ( $user->is_superuser || $user->belongs_to_body( $b->id ) ) ) {
+        $where = {
+            body_id => $where->{body_id},
+            -or => { send_method => [ { '!=' => 'Triage' }, undef ] },
```

Comment:
I think this can just be ` send_method => [ { '!=' => 'Triage' }, undef ]` - not sure what the `-or` wrapper is for.